### PR TITLE
Visually separated the list of control values from the example.

### PR DESCRIPTION
**Pull request by hepcat72**
This will aid people who are quickly skimming the readme. There wasn't anything wrong in the doc. Everything would be clear if the reader is reading thoughtfully. But if they're skimming and looking for what to supply to the node for the first time, their eye is drawn to the JSON example.  Since many nodes accept JSON input, and supplying `{"state": "off"}`, `{"state": 0}`, or `{"state": 1}` works, it might take them awhile to realize their oversight.

- [x] Bugfix (non-breaking *documentation* change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red-nodes/blob/master/CONTRIBUTING.md)
- [x] For non-bugfix PRs, I have discussed this change in another issue.
- [ ] I have run `grunt` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

File: hardware/wemo/README.md [hunk 1]
```diff
@@ -21,10 +21,14 @@ from opening the config dialog and selecting the node you want.
...
- * Strings on/off
- * integers 1/0
- * boolean true/false
- * an Object like this (lights only & color control is still work in the progress) 
+ * A single value, such as
+     * Strings `on`/`off`
+     * integers `1`/`0`
+     * boolean `true`/`false`
+```
+on
```
Comment (dceejay) [inline hardware/wemo/README.md]: on ? or did you mean or ?

Comment (hepcat72) [reply hardware/wemo/README.md]: It's an example of what to send to the node, so `on`. I admit it's weird that there are sub-items between the item preceding the example. Let me see if I can improve it...

Comment (dceejay) [reply hardware/wemo/README.md]: ok - so it should be titled "object"  (like integer boolean etc) - then have the example.

Comment (dceejay) [reply hardware/wemo/README.md]: actually - or without the blockquotes would work fine.

Also while you are at it... - just before the list maybe make the line
The node accecpts the following inputs   - into - The node accepts the following `msg.payload` inputs

**Pull request merged**